EventFilter/OnlineMetaDataRawToDigi/python/OnlineMetaDataRawToDigi_cfi.py

```diff
@@ -0,0 +1,2 @@
+import FWCore.ParameterSet.Config as cms
+import EventFilter.OnlineMetaDataRawToDigi.onlineMetaDataRawToDigi_cfi
```

Comment:
please remove this file.
It is rather confusing to have both onlineMetaDataRawToDigi_cfi and OnlineMetaDataRawToDigi_cfi different only in the first letter capitalization